lib/rubocop/cli.rb

```diff
@@ -79,7 +79,6 @@ def execute_runners(paths)
             run_line_length_cop_auto_gen_config(paths)
           else
             puts Rainbow(SKIPPED_PHASE_1).yellow
-            ''
```

Comment:
This changes the value assigned to `line_length_contents` from an empty string to `nil`. I have no idea if that matters or not.